src/pages/submit.js

```diff
@@ -1,10 +1,210 @@
-  <Layout>
-    <h1>This is the submit work page!</h1>
-  </Layout>
-);
+import * as React from "react";
+import { graphql, useStaticQuery } from "gatsby";
+import Layout from "../components/Layout";
+import { H2, YELLOW, MarginContainer } from "../styles/styles";
+import { PrismicRichText } from "@prismicio/react";
+import {
+  HeaderImage,
+  GetInvolvedSection,
+  QuestionsSection,
+  QuestionsText,
+  GetInvolvedText,
+  ContentContainer,
+  InformationContainer,
+  FormContainer,
+  FormRow,
+  Input,
+  TextArea,
+  SubmitDescription,
+  HiddenInput,
+  UploadButton,
+  UploadButtonText,
+  UploadIcon,
+  PaddedInput,
+  SubmitContainer,
+} from "../styles/submit.styles";
+import Button from "../components/Button";
+
+const Submit = () => {
+  const submitQuery = useStaticQuery(graphql`
+    query submitQuery {
+      prismicSubmitWorkPage {
+        data {
+          call_to_action {
+            richText
+          }
+          credits_label {
+            text
+          }
+          email_label {
+            text
+          }
+          equipment_label {
+            text
+          }
+          get_involved {
+            richText
+          }
+          header_image {
+            url
+            alt
+          }
+          name_label {
+            text
+          }
+          project_description_label {
+            text
+          }
+          project_medium_label {
+            text
+          }
+          project_title_label {
+            text
+          }
+          questions {
+            richText
+          }
+          submit_button_label {
+            text
+          }
+          submit_intro {
+            richText
+          }
+          submit_title {
+            text
+          }
+          upload_button_label {
+            text
+          }
+        }
+      }
+    }
+  `);
+
+  const {
+    prismicSubmitWorkPage: {
+      data: {
+        call_to_action: callToAction,
+        credits_label: creditsLabel,
+        email_label: emailLabel,
+        equipment_label: equipmentLabel,
+        get_involved: getInvolved,
+        header_image: headerImage,
+        name_label: nameLabel,
+        project_description_label: projectDescriptionLabel,
+        project_medium_label: projectMediumLabel,
+        project_title_label: projectTitleLabel,
+        questions: questions,
+        submit_button_label: submitButtonLabel,
+        submit_intro: submitIntro,
+        submit_title: submitTitle,
+        upload_button_label: uploadButtonLabel,
+      },
+    },
+  } = submitQuery;
+
+  return (
+    <Layout>
+      <HeaderImage src={headerImage.url} alt={headerImage.alt} />
+      <MarginContainer>
+        <ContentContainer>
+          <InformationContainer>
+            <GetInvolvedSection>
+              <PrismicRichText
+                field={getInvolved.richText}
+                components={{
+                  paragraph: ({ children }) => (
+                    <GetInvolvedText>{children}</GetInvolvedText>
```

Comment:
Since we also want to include `<H3>` in this section, you could turn `<GetInvolvedText>` into a div that wraps the `<PrismicRichText>` component, and in the components prop just return a simple `<P>` and `<H3>`